src/components/Navbar/Navbar.types.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  ChainSelectorProps,
+  ChainSelectori18n,
+} from "../ChainSelector/ChainSelector.types"
+import { UserMenuI18N, UserMenuProps } from "../UserMenu/UserMenu.types"
+
+enum NavbarPages {
+  MARKETPLACE = "marketplace",
+  CREATE = "create",
+  EXPLORE = "explore",
+  LEARN = "learn",
+  GOVERNANCE = "governance",
+}
+
+enum NavbarPagesUrls {
```

Comment:
These pages should be loaded through the environment variables, would you mind using it through the env package?